testfm/constants.py

```diff
@@ -1,17 +1,25 @@
-import configparser
+from dynaconf import LazySettings
 
-config = configparser.ConfigParser()
-config.read("testfm.properties")
+settings = LazySettings(
+    envvar_prefix="TesTFM",
+    core_loaders=["YAML"],
+    settings_file=["settings.yaml"],
+    preload=["*.yaml"],
+    includes=["settings.local.yaml", ".secrets.yaml", ".secrets_*.yaml"],
+    envless_mode=True,
+    lowercase_read=True,
+)
+
+RHN_USERNAME = settings.subscription.RHN_USERNAME
+RHN_PASSWORD = settings.subscription.RHN_PASSWORD
+FM_RHN_POOLID = settings.subscription.FM_RHN_POOLID
+DOGFOOD_ORG = settings.subscription.DOGFOOD_ORG
+DOGFOOD_ACTIVATIONKEY = settings.subscription.DOGFOOD_ACTIVATIONKEY
+CAPSULE_DOGFOOD_ACTIVATIONKEY = settings.subscription.CAPSULE_DOGFOOD_ACTIVATIONKEY
+DOGFOOD_URL = settings.subscription.DOGFOOD_URL
+HOTFIX_URL = settings.URLS.HOTFIX_URL
+SERVER_HOSTNAME = settings.SERVER.SERVER_HOSTNAME
```

Comment:
You need not to redefine them here unless you want to modify them to new value.

So, directly use these settings in a test without defining them as constants (Those are not constants as well).

Reply:
We in testfm don't update/modify these values for tests, so that's why it's defined here as constants, and we can consider using these settings in tests as enhancements. currently thinking to move to dynaconf with minimal changes

Reply:
ACK.